src/Juvix/Compiler/Abstract/Translation/FromConcrete.hs

```diff
@@ -428,22 +433,28 @@ goUniverse = id
 
 goFunction :: (Members '[Error ScoperError, InfoTableBuilder] r) => Function 'Scoped -> Sem r Abstract.Function
 goFunction (Function l r) = do
-  _funParameter <- goFunctionParameter l
-  _funReturn <- goExpression r
-  return Abstract.Function {..}
+  params <- goFunctionParameters l
+  ret <- goExpression r
+  return $
+    Abstract.Function (NonEmpty.head params) $
```

Comment:
`NonEmpty.head` is exported by `Juvix.Prelude`

Reply:
Done